packages/framework/resources/views/components/markdown-heading.blade.php

```diff
@@ -0,0 +1,13 @@
+@props(['level' => 1, 'id' => null, 'addPermalink' => config('markdown.features.permalinks', true)])
+
+@php
+    $tag = 'h' . $level;
+    $id = $id ?? \Illuminate\Support\Str::slug($slot);
+@endphp
+
+<{{ $tag }} {{ $attributes->merge(['id' => $id]) }}>
+    {!! $slot !!}
```

Comment:
Confirmed this does not open up for XSS.

Example input:

```md
# Quickstart Guide <script>console.log('Quickstart Guide');</script> <strong>bold</strong> <em>italic</em>
```

```html
<h1 id="quickstart-guide-ltscriptconsolelogquickstart-guideltscript-strongboldstrong-emitalicem">
    Quickstart Guide &lt;script&gt;console.log('Quickstart Guide');&lt;/script&gt; <strong>bold</strong> <em>italic</em>
            <a href="#quickstart-guide-ltscriptconsolelogquickstart-guideltscript-strongboldstrong-emitalicem" class="heading-permalink" aria-label="Permalink for this section"></a>
    </h1>
```

<img width="941" alt="image" src="https://github.com/user-attachments/assets/79dcda1d-7cd4-4e42-b3e9-21f17ea99b1e">

Reply:
And tested in https://github.com/hydephp/develop/commit/43d82a4327dcf2cd68d02a97cec041bdcd8dbe28